review parcial 2

pardonilintuclan opened the pull request.

pardonilintuclan committed:
Add files via upload

pardonilintuclan committed:
Delete Diagrama Contenedores.drawio.pdf

pardonilintuclan committed:
Delete Diagrama de Componentes.drawio.pdf

pardonilintuclan committed:
Delete Diagrama de Contexto.drawio.pdf

pardonilintuclan committed:
Delete Proyecto Informatico.docx

pardonilintuclan committed:
primer pull request proyecto

pardonilintuclan committed:
Creación carpeta parcial 1 y  el readme base 

por favor revisar y agregar cambios

gustavogelpud committed:
Actualización de Historias de Usuario

Adicione 1 historia para el registro del usuario, 1 en olvidé mi contraseña y revise las demás historias por si hacia falta alguna importante.

juanforero5 committed:
App login screen requirements EN

Documento de requerimentos EN

gustavogelpud committed:
Adición versión en español.

Se actualizó la versión del Readme adicionando unos nuevos criterios y la versión en español. Quedaron ambas, Ingles y Español.

juanforero5 committed:
Diagrama de componentes C4

juanforero5 committed:
Carpeta repo codigo

gustavogelpud committed:
Se adiciona el Diagrama de Contexto.

Quedo atento a cualquier comentario muchachos.

gustavogelpud committed:
Delete Diagrama Contexto

gustavogelpud committed:
Se carga el nuevo Diagrama de Contexto

gustavogelpud committed:
Delete Diagrama de componentes

Se borra el diagrama por que el correcto es el de Contenedores.

gustavogelpud committed:
Se sube el correcto Diagrama de Contenedores.

pardonilintuclan committed:
Add files via upload

la v2 que encargo mauro subir

gustavogelpud committed:
Se carga el Diagrama de Componentes

UAOMauricio committed:
Update Readmeentrega parcial1.txt

UAOMauricio committed:
Update login.html

pardonilintuclan committed:
Create login_success.html

pardonilintuclan committed:
Add files via upload

v3 funcionamiento cuestionable pero es lo mejor que se pudo hacer

pardonilintuclan committed:
Update Readmeentrega parcial1.txt

bfmora commented:
Buen trabajo en general un par de observaciones iniciales:

¿Porque hay diferentes carpetas con versiones?
¿Que IDE usaron para programar la solución?
¿Que estándar o definición de buenas practicas usaron para la organización del código?
¿Porque usaron las tecnologías que tienen definidas en el proyecto?
¿Como se dividieron la participación en el proyecto entre los integrantes?

bfmora commented:
Muchachos la misma recomendación que se hizo sobre PHP durante la clase, para que la tengan en cuenta.

Calificación 4,5.

UAOMauricio committed:
Código Inicial de desarrollo

pardonilintuclan committed:
Delete CORTE_2 directory

First Lenguage Correction and Directory order

pardonilintuclan committed:
Delete Parcial1 directory

Second Lenguaje Correction and Order Files

pardonilintuclan committed:
Delete Diagrama de Contexto.drawio.pdf

pardonilintuclan committed:
Delete Diagrama de Componentes.drawio.pdf

pardonilintuclan committed:
Delete Diagrama Contenedores.drawio.pdf

pardonilintuclan committed:
Delete README.md

pardonilintuclan committed:
Ultra Lenguaje update repository and files order

An attempt was made to leave the entire repository in a single language without altering the language in versions prior to v5 to leave a record of the evolution of the project.

pardonilintuclan committed:
second update readme first commit

only creation of readme update later

UAOMauricio committed:
Update login.php

Changes made

juanforero5 committed:
EN: Carga diagramas de contexto y componentes

pardonilintuclan committed:
V5

need help

gustavogelpud committed:
ATAM document upload

I remain attentive to any adjustments that must be made.

gustavogelpud committed:
Delete Evaluación de Arquitectura ATAM.docx

Delete file by name in Spanish.

gustavogelpud committed:
ATAM document upload

Upload file name in English

gustavogelpud committed:
QAW document upload

Load QAW document file.

pardonilintuclan committed:
Add files via upload

bfmora commented:
- Muchachos no le hicieron merge a la primera revisión para poder ver solo la revisión del entregable 2.
- No me enviaron el correo para la revisión.
- Para el manejo de las conexiones de las base de datos les recomiendo usar un global helper para poder mantener la conexión en un solo archivo.


Calificación **4.0**
